odl/operator/operator.py

```diff
@@ -470,10 +470,10 @@ def __init__(self, domain, range, linear=False):
             raise TypeError('`range` {!r} is not a `Set` instance'
                             ''.format(range))
 
-        self._domain = domain
-        self._range = range
-        self._is_linear = bool(linear)
-        self._is_functional = isinstance(range, Field)
+        self.__domain = domain
+        self.__range = range
+        self.__is_linear = bool(linear)
+        self.__is_functional = isinstance(range, Field)
```

Comment:
Perhaps a comment here that we cache this for efficiency, otherwise people (including me) will ask this over and again.